_aggregations/bucket/children.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: default
+title: Children
+parent: Bucket aggregations
+grand_parent: Aggregations
+nav_order: 15
+---
+
+# Children
+
+The `children` aggregation connects parent documents with their related child documents. This allows you to analyze relationships between different types of data in a single query, rather than having to run multiple queries and combine the results yourself.
+
+---
+
+## Example index, sample data, and children aggregation query
+
+For example, if you have a parent-child relationship between authors, posts, and comments, you can analyze the relationships between the data different types (`authors`, `posts`, and `comments`) in a single query, without having to run multiple queries and combine the results manually. 
+
+The `authors` aggregation groups the documents by the `author.keyword` field. Within each author group, we have a `children` aggregation that looks at the associated posts. Inside the `posts` aggregation, another `children` aggregation looks at the comments associated with each post. Within the `comments` aggregation, the `value_count` aggregation counts the number of comments for each post.
+
```

Comment:
Above: Please replace both instances of the phrasal verb "looks at" with a more precise verb. What do the aggregations actually do?